### PR TITLE
create modal view in consultation table

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Create modal view in consultation table in queues-in-progress

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]
El desarrollo es correcto

File: .storybook/main.ts [hunk 1]
```diff
@@ -17,5 +11,12 @@ const config: StorybookConfig = {
     autodocs: true,
     defaultName: "Documentation",
   },
+  async viteFinal(config) {
```
Comment (carin91) [inline .storybook/main.ts:14]: why do we need this change?

File: src/components/feedback/InteractiveModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import { StyledContainer, StyledModal, StyledModalFields } from "./styles";
+
+interface InteractiveModalProps {
+  portalId: string;
+  title: string;
+  onCloseModal: () => void;
+  infoData: IActions;
+  labels: ILabel[];
+  handleReprocess: () => void;
+  actions?: IAction[];
```
Comment (carin91) [inline src/components/feedback/InteractiveModal/index.tsx:23]: This prop is not in use, can you delete it?

File: src/components/feedback/InteractiveModal/styles.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+const StyledContainer = styled.div`
+  position: relative;
+
+  div {
+    z-index: 3;
+  }
+`;
+
+const StyledModal = styled.div<IStyledModal>`
+  background-color: ${inube.palette.neutral.N10};
```
Comment (carin91) [inline src/components/feedback/InteractiveModal/styles.ts:17]: Define a theme validation, for future implementation of a theme provider.

File: src/pages/queues/outlets/queuesInProgress/components/DetailsModal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { Icon } from "@inubekit/icon";
+import { MdOutlineRemoveRedEye } from "react-icons/md";
+
+import { InteractiveModal } from "@components/feedback/InteractiveModal";
+import { IActions } from "@components/data/Table/props";
+import { labelsModal } from "../../config/table";
+
+interface IDetailsModalProps {
```
Comment (carin91) [inline src/pages/queues/outlets/queuesInProgress/components/DetailsModal/index.tsx:9]: is it necessary? the DetailsModal has just 1 prop, however, you have an interface + destructuring. It is good to hold a standard to improve consistency, however, it is important to write code as simply as possible. Reevaluate this solution.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: .storybook/main.ts (hunk 1 above)
Comment (ANGIECATHERINEP) [reply .storybook/main.ts]: resolved

File: src/components/feedback/InteractiveModal/index.tsx (hunk 2 above)
Comment (ANGIECATHERINEP) [reply src/components/feedback/InteractiveModal/index.tsx]: resolved

File: src/components/feedback/InteractiveModal/styles.ts (hunk 3 above)
Comment (ANGIECATHERINEP) [reply src/components/feedback/InteractiveModal/styles.ts]: resolved

File: src/pages/queues/outlets/queuesInProgress/components/DetailsModal/index.tsx (hunk 4 above)
Comment (ANGIECATHERINEP) [reply src/pages/queues/outlets/queuesInProgress/components/DetailsModal/index.tsx]: resolved

**Review by carin91 — APPROVED** [review]

**Pull request merged**